google/cloud/bigtable/internal/bulk_mutator.cc

```diff
@@ -79,7 +79,7 @@ google::bigtable::v2::MutateRowsRequest const& BulkMutatorState::BeforeStart() {
 }
 
 std::vector<int> BulkMutatorState::OnRead(
-    google::bigtable::v2::MutateRowsResponse& response) {
+    google::bigtable::v2::MutateRowsResponse&& response) {
```

Comment:
Should all these simply be `s/&&//` instead?

Reply:
After discussing further with @devbww, I am no longer convinced that using `&&` is much of a constraint.

So changed `OnRead` to pass by value, which is recommended by the style guide and more consistent with the rest of the library.